src/host_beta/messageRouter.ts

```diff
@@ -70,6 +71,10 @@ export class MessageRouter {
                 const [url, line, column, ignoreTabChanges] = args;
                 this.openInEditor(url, line, column, ignoreTabChanges);
                 return true;
+            case 'cssMirrorContent':
+                const [cssUrl = url, newContent] = args;
```

Comment:
why is `url` needed here?

Reply:
`url` is the parameter name within `args`. I had to rename it to `cssUrl` because the compiler doesn't like that we're redeclaring the `url` const from the `openInEditor` case.